config.lua.dist

```diff
@@ -120,6 +120,7 @@ bosstiaryKillMultiplier = 1
 boostedBossSlot = true
 boostedBossLootBonus = 250
 boostedBossKillBonus = 3
+rateCharmShopPrice = 1.0
```

Comment:
It is better to rename this variable and move it to just below the: "bosstiaryKillMultiplier "
Which will also require you to rename it in configmanager and elsewhere.

```suggestion
bestiaryRateCharmShopPrice = 1.0
```

Reply:
I'm on it